examples/tree/plot_unveil_tree_structure.py

```diff
@@ -148,6 +159,8 @@
 
 ##############################################################################
 # We can compare the above output to the plot of the decision tree.
+# Here, we show the proportions of samples of each class that reach each
+# node corresponding to the actual elements of `tree_.value` array.
 
 tree.plot_tree(clf)
```

Comment:
You still have to set `proportion=True` to match the text.

Reply:
Ah right, I reverted it when I added the deprecation, but forgot to unrevert when I un-reverted the deprecation